corehq/apps/fixtures/resources/v0_6.py

```diff
@@ -0,0 +1,7 @@
+from . import v0_1
+
+
+class LookupTableItemResource(v0_1.LookupTableItemResource):
+
+    class Meta(v0_1.LookupTableItemResource.Meta):
+        always_return_data = True
```

Comment:
So this changed the response to include the fixture data as well?
Did this also change the response code to 200 or that was something else?

Reply:
Yes, it changed both. It changed the 204 ("no content") response to 200 because now the response has content.

Reply:
Link to docs for reference: https://django-tastypie.readthedocs.io/en/latest/resources.html#always-return-data